src/test/java/org/jabref/logic/formatter/bibtexfields/UnicodeToLatexFormatterTest.java

```diff
@@ -14,23 +18,21 @@ void setUp() {
         formatter = new UnicodeToLatexFormatter();
     }
 
-    @Test
-    void formatWithoutUnicodeCharactersReturnsSameString() {
-        assertEquals("abc", formatter.format("abc"));
-    }
-
-    @Test
-    void formatMultipleUnicodeCharacters() {
-        assertEquals("{{\\aa}}{\\\"{a}}{\\\"{o}}", formatter.format("\u00E5\u00E4\u00F6"));
+    private static Stream<Arguments> testCases() {
+        return Stream.of(
+                Arguments.of("", ""), // empty string input
+                Arguments.of("abc", "abc"), // non unicode input
+                Arguments.of("\u00E5\u00E4\u00F6", "{{\\aa}}{\\\"{a}}{\\\"{o}}"), // multiple unicodes input
+                Arguments.of("\u0081", ""), // high code point unicode, boundary case: cp = 129
+                Arguments.of("\u0080", ""), // high code point unicode, boundary case: cp = 128 < 129
+                Arguments.of(new UnicodeToLatexFormatter().getExampleInput(), "M{\\\"{o}}nch")
```

Comment:
Please switch parameters to comply with `assertEquals` semantics: first the `expected`, then the `input` value.

Reply:
Code is updated as suggested. :)